steering_committee.md

```diff
@@ -1,38 +1,12 @@
-# Organization Structure of OpenIntegrationEngine project
+# Steering Committee of OpenIntegrationEngine project
 
 ## Current members
 
-### Steering Committee
-
-#### As of 31st of March 2026
-
 - Chris Gibson
-- Jonathan Bartels
-- Kaur Palang
-- Kiran Ayyagari
-- Paul Coyne
+- [Jonathan Bartels](https://github.com/jonbartels)
+- [Kaur Palang](https://github.com/kpalang)
+- [Kiran Ayyagari](https://github.com/kayyagari)
+- [Kaur Palang](https://github.com/kpalang)
 - Sean Rowe
-- Tony Germano
-
-## Maintainers
-
-#### As of 26th of March 2025
-
-- Jonathan Bartels
-- Kaur Palang
-- Paul Coyne
-- Tony Germano
-
----
-
-## Historical compositions
-
-### Steering Committee
-
-#### As of 26th of March 2025
-
-- Jonathan Bartels
-- Kaur Palang
-- Paul Coyne
-- Tony Germano
+- [Kaur Palang](https://github.com/kpalang)
```

Comment:
- Kaur listed 3 times
- Paul and I were removed
- Chris Gibson's github is https://github.com/gibson9583
- Sean Rowe's github is https://github.com/ssrowe